services/minespace-web/src/components/Forms/projects/projectSummary/ProjectSummaryFileUpload.js

```diff
@@ -11,19 +17,188 @@ const propTypes = {
   params: PropTypes.objectOf(PropTypes.string).isRequired,
 };
 
-export const ProjectSummaryFileUpload = (props) => (
-  <Field
-    id="fileUpload"
-    name="fileUpload"
-    component={FileUpload}
-    uploadUrl={PROJECT_SUMMARY_DOCUMENTS(props.params)}
-    acceptedFileTypesMap={props.acceptedFileTypesMap}
-    onFileLoad={props.onFileLoad}
-    onRemoveFile={props.onRemoveFile}
-    allowRevert
-    allowMultiple
-  />
-);
+const notificationDisabledStatusCodes = [409]; // Define the notification disabled status codes
+
+export const ProjectSummaryFileUpload = (props) => {
+  const [archivedFileModalMessage, setArchivedFileModalMessage] = useState("");
+  const [isArchivedFileModalVisible, setIsArchivedFileModalVisible] = useState(false);
+  const [relaplaceableFileModalMessage, setRelaplaceableFileModalMessage] = useState("");
+  const [isRelaplaceableFileModalVisible, setRelaplaceableFileModalVisible] = useState(false);
+  const [fileName, setFileName] = useState("");
+  const [shouldReplaceFile, setShouldReplaceFile] = useState(false);
+  const [replaceFileUploadUrl, setReplaceFileUploadUrl] = useState(false);
+  const [mineDocumentGuid, setMineDocumentGuid] = useState(null);
+  const [mineGuid, setMineGuid] = useState(null);
+  const [fileDetails, setFileDetails] = useState(null);
+  const [shouldAbortUpload, setShouldAbortUpload] = useState(false);
+
+  const userInfo = useSelector(getUserInfo);
+
+  const handleCloseModal = () => {
+    setIsArchivedFileModalVisible(false);
+    setRelaplaceableFileModalVisible(false);
+    setShouldAbortUpload(true);
+  };
+
+  const childRef = useRef(null);
```

Comment:
Is childRef used anywhere?

Reply:
Nice catch. No it is not used so removed.